### PR TITLE
added reference to the Group owner field

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#1248

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/deploy-an-isolated-sharepoint-online-team-site.md [hunk 1]
```diff
@@ -163,12 +163,17 @@ First, create the SharePoint Online team site with these steps.
 4. On the **Create a site** page, click **Team site**.
     
 5. In **Site name**, type a name for the team site. 
+
+6. In **Group owner**, specify the owner of the associated Office 365 Group.
+
+> [!NOTE]
+> Group owner is automatically included as site admin, therefore can manage group membership and have full access to the site.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/deploy-an-isolated-sharepoint-online-team-site.md]:
```suggestion
> Group owner is automatically included as a site admin and can therefore manage group membership and have full access to the site. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only a suggestion.

**Comment by MikePlumleyMSFT** [comment]
The issue actually is that the procedure should *not* include creating an Office 365 group. I have a separate PR in the queue to update the procedure.

**Pull request closed without merging**